heat-stack/model/Homes.tsx

```diff
@@ -0,0 +1,12 @@
+export interface Home {
+	livingArea: number
+	fuelType: string // number?
```

Comment:
`suggestion`: the database won't have a concept of it, but I might use an [enum ](https://www.typescriptlang.org/docs/handbook/enums.html) here.

Reply:
I think we can do 
`fuelType: 'Natural Gas' | 'Oil'`

Reply:
@alanisaac should there be other options?

Reply:
The options in the spreadsheet are:

* Natural Gas
* Fuel Oil
* Propane

There is a calculation distinction between fuel oil and propane.

Reply:
updated to `fuelType: 'Natural Gas' | 'Oil' | 'Propane'`